tensorflow/python/feature_column/feature_column_test.py

```diff
@@ -3213,12 +3213,13 @@ def test_transform_with_weighted_column(self):
     weights = fc.weighted_categorical_column(ids, 'weights')
     indicator = fc.indicator_column(weights)
     features = {
-      'ids': constant_op.constant(['c', 'b', 'a'], shape=(1, 3)),
-      'weights': constant_op.constant([2., 4., 6.], shape=(1, 3))
+      # Github issue 12583
+      'ids': constant_op.constant([['c', 'b', 'unknown']]),
```

Comment:
please, add a specific test for unknown instead of changing this test.
also please verify unknown without weight column too.

Reply:
sure, I'll add it later.

Reply:
Hi, test cases has been added as suggested.